public/pages/Destinations/components/createDestinations/CustomWebhook/validate.js

```diff
@@ -16,35 +16,45 @@
 import { URL_TYPE } from '../../../containers/CreateDestination/utils/constants';
 
 export const validateUrl = (value, allValues) => {
-  const type = allValues.type;
+  // type is "http" when the component 'URLInfo' is used to define a monitor
+  const type = allValues.type ? allValues.type : 'http';
   if (allValues[type].urlType !== URL_TYPE.FULL_URL) return;
   if (!value) return 'Required';
   const regname = '((www\\.)?[-a-zA-Z0-9@:%._\\+~#=]{2,256}\\.[a-z]{2,63})';
-  const ipv4 = '(((25[0-5]|(2[0-4]|1{0,1}[0-9]){0,1}[0-9])\\.){3}(25[0-5]|(2[0-4]|1{0,1}[0-9]){0,1}[0-9]))';
+  const ipv4 =
+    '(((25[0-5]|(2[0-4]|1{0,1}[0-9]){0,1}[0-9])\\.){3}(25[0-5]|(2[0-4]|1{0,1}[0-9]){0,1}[0-9]))';
   const h16 = '([0-9a-fA-F]{1,4})';
   const ls32 = `((${h16}:${h16})|${ipv4})`;
-  const ipv6 = `\\[(`+
-    `((${h16}:){6}${ls32})|`+
-    `(::(${h16}:){5}${ls32})|`+
-    `(${h16}?::(${h16}:){4}${ls32})|`+
-    `(((${h16}:){0,1}${h16})?::(${h16}:){3}${ls32})|`+
-    `(((${h16}:){0,2}${h16})?::(${h16}:){2}${ls32})|`+
-    `(((${h16}:){0,3}${h16})?::${h16}:${ls32})|`+
-    `(((${h16}:){0,4}${h16})?::${ls32})|`+
-    `(((${h16}:){0,5}${h16})?::${h16})|`+
-    `((${h16}:){0,6}${h16})?::`+
+  const ipv6 =
+    `\\[(` +
+    `((${h16}:){6}${ls32})|` +
+    `(::(${h16}:){5}${ls32})|` +
+    `(${h16}?::(${h16}:){4}${ls32})|` +
+    `(((${h16}:){0,1}${h16})?::(${h16}:){3}${ls32})|` +
+    `(((${h16}:){0,2}${h16})?::(${h16}:){2}${ls32})|` +
+    `(((${h16}:){0,3}${h16})?::${h16}:${ls32})|` +
+    `(((${h16}:){0,4}${h16})?::${ls32})|` +
+    `(((${h16}:){0,5}${h16})?::${h16})|` +
+    `((${h16}:){0,6}${h16})?::` +
```

Comment:
These lines of changes are made by the linter

Reply:
Could you verify this works as expected ? i.e verify all tests are passing.